core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/event/span/export/optel/OpentelemetrySpanExporterManager.java

```diff
@@ -23,8 +23,8 @@
  */
 public class OpentelemetrySpanExporterManager implements InternalSpanExportManager<EventContext> {
 
-  private static final OpenetelemetryCoreEventInternalSpanExporterFactory SPAN_EXPORTER_FACTORY =
-      getOpenetelemetryCoreEventInternalSpanExporterFactory();
+  private static final OpentelemetryCoreEventInternalSpanExporterFactory SPAN_EXPORTER_FACTORY =
```

Comment:
Shouldn't it be OpenTelemetry... ? Both here as in getOpentelemetryCoreEventInternalSpanExporterFactory